src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.spec.ts

```diff
@@ -1,16 +1,35 @@
+import { Component, Input } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
+
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import { NgxPageScrollCoreModule } from 'ngx-page-scroll-core';
+
+import { TemplateSession } from '../../../../services/feedback-sessions.service';
+import { Course } from '../../../../types/api-output';
+import {
+  SessionEditFormMode,
+  SessionEditFormModel,
+} from '../../../components/session-edit-form/session-edit-form-model';
 import { InstructorHelpSessionsSectionComponent } from './instructor-help-sessions-section.component';
 
+@Component({ selector: 'tm-example-box', template: '' })
+class ExampleBoxStubComponent {}
+@Component({ selector: 'tm-session-edit-form', template: '' })
+class SessionEditFormStubComponent {
+  @Input() formMode: SessionEditFormMode = SessionEditFormMode.ADD;
+  @Input() model: SessionEditFormModel = {} as SessionEditFormModel;
+  @Input() courseCandidates: Course[] = [];
+  @Input() templateSessions: TemplateSession[] = [];
+}
+
 describe('InstructorHelpSessionsSectionComponent', () => {
   let component: InstructorHelpSessionsSectionComponent;
   let fixture: ComponentFixture<InstructorHelpSessionsSectionComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [InstructorHelpSessionsSectionComponent],
+      declarations: [InstructorHelpSessionsSectionComponent, ExampleBoxStubComponent, SessionEditFormStubComponent],
       imports: [NgbModule, RouterTestingModule, NgxPageScrollCoreModule],
```

Comment:
If you create those stub just to make the test passing, I think you can import `SessionEditFormModule` here to get it passed.

Reply:
I tried to import this module. But i received a `NullInjectorError: No provider for MatSnackBar!` So i made the stub to pass the test. Not ideal, but any ideas on the error?

![image](https://user-images.githubusercontent.com/28994607/82276421-ce741380-99b7-11ea-9edf-4f422b177844.png)

Reply:
See how existing test deal with this? (try search `MatSnackBarModule` and look for files end with `spec.ts`)

Reply:
Okay, found the error. was importing the `MatSnackBarModule` from `@angular/material/snack-bar` instead of `@angular/material`. Thanks